File: Python/백준/Bronze/5575. 타임 카드/타임 카드.py
for i in range(3):
    aa = list(map(int,input().split()))
    time1 = aa[0]*3600+aa[1]*60+aa[2]
    time2 = aa[3]*3600+aa[4]*60+aa[5]
    time = time2 - time1
    print(time//3600,(time-(time//3600)*3600)//60,time%60)

File: C#/백준/Bronze/5575. 타임 카드/타임 카드.cs
using System;

namespace q{
    class w{
        static void Main(){
            for (int i=0; i<3; i++){
                int[] a = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int t1 = a[0]*3600+a[1]*60+a[2];
                int t2 = a[3]*3600+a[4]*60+a[5];
                int time = t2-t1;
                Console.WriteLine($"{time/3600} {(time%3600)/60} {time%60}");
            }
        }
    }
}